common/src/main/java/me/shedaniel/architectury/event/events/GuiEvent.java

```diff
@@ -32,62 +32,131 @@
 
 import java.util.List;
 
+/**
+ * Should be moved to the client package and renamed to ScreenEvent in version 2.0
+ */
 @Environment(EnvType.CLIENT)
 public interface GuiEvent {
     /**
-     * Invoked after in-game hud is rendered, equivalent to forge's {@code RenderGameOverlayEvent.Post@ElementType#ALL} and fabric's {@code HudRenderCallback}.
+     * @see RenderHud#renderHud(PoseStack, float)
      */
     Event<RenderHud> RENDER_HUD = EventFactory.createLoop();
+    /**
+     * @see DebugText#gatherText(List)
+     */
     Event<DebugText> DEBUG_TEXT_LEFT = EventFactory.createLoop();
     Event<DebugText> DEBUG_TEXT_RIGHT = EventFactory.createLoop();
     /**
-     * Invoked during Screen#init after previous widgets are cleared, equivalent to forge's {@code GuiScreenEvent.InitGuiEvent.Pre}.
+     * @see ScreenInitPre#init(Screen, List, List)
      */
     Event<ScreenInitPre> INIT_PRE = EventFactory.createInteractionResult();
     /**
-     * Invoked after Screen#init, equivalent to forge's {@code GuiScreenEvent.InitGuiEvent.Post}.
+     * @see ScreenInitPost#init(Screen, List, List)
      */
     Event<ScreenInitPost> INIT_POST = EventFactory.createLoop();
+    /**
+     * @see ScreenRenderPre#render(Screen, PoseStack, int, int, float)
+     */
     Event<ScreenRenderPre> RENDER_PRE = EventFactory.createInteractionResult();
+    /**
+     * @see ScreenRenderPost#render(Screen, PoseStack, int, int, float)
+     */
     Event<ScreenRenderPost> RENDER_POST = EventFactory.createLoop();
-    
     /**
-     * Invoked during Minecraft#setScreen, equivalent to forge's {@code GuiOpenEvent}.
+     * @see SetScreen#modifyScreen(Screen)
      */
     Event<SetScreen> SET_SCREEN = EventFactory.createInteractionResultHolder();
     
     @Environment(EnvType.CLIENT)
     interface RenderHud {
+        /**
+         * Called after the in-game hud is rendered.
+         * Equal to the forge {@code RenderGameOverlayEvent.Post@ElementType#ALL} event and fabric's {@code HudRenderCallback}.
+         * 
+         * @param matrices The render buffer.
+         * @param tickDelta The delta tick.
```

Comment:
"Tick delta" would be better than "delta tick" here, but you could also consider using a more explanative description.

Reply:
As Yarn puts it, "progress for linearly interpolating between the previous and current game state" (basically the delta parameter to `MathHelper.lerp` or whatever Mojang's mappings call it).